CONTRIBUTING.md

```diff
@@ -0,0 +1,197 @@
+- **Technical contributors**
+    - Contributors writing HTML, CSS, or Javascript for the website that modifies the way 
+      it looks or behaves.
+
+In this guide, we start off by explaining contribution steps that apply
+to both kinds of contributors. Then, the guide branches off to explain specifics for each.
+
+## Documentation tools
+
+conda.org is built with [Docusaurus 2][docusaurus], a modern static website generator that 
+uses a friendly Markdown format to create the website copy. If you are not familiar with 
+Markdown, check out the [Markdown Guide project][markdown-guide].
+
+## Working on issues
+
+The conda.org project has an [issue board][issue-board] from which technical contributors can choose issues. If there is an aspect of the website that you feel needs work, please create a new issue using the **Add item** button.
+
+To be assigned to an existing issue, comment on it and one of the maintainers will ensure you are assigned.
+
+## Github flow
+
+### Prerequisites
+
+To follow this guide, you will need to have these programs installed on your computer:
+
+- git: [installation link][installing-git]
+
+It is also highly encouraged that you at least have a basic understanding of how to use
+git at the command line. If you do not, please check out the book [Pro Git][pro-git],
+available for free from the official git website.
+
+### Github flow
+
+#### 1. Creating a fork
+
+The very first step to contributing is creating your own fork of the 
+[conda-dot-org repository][conda-dot-org-repo]. If you have never done this before, 
+we strongly encourage you to read the 
+[Fork a repo][github-forking-how-to] guide from GitHub first.
+
+#### 2. Clone your repository
+
+After you have created your fork, you are ready to clone this repository to your 
+local computer, where you can begin making edits. The commands for doing so will 
+be different depending on your user name, but a typical example is shown below:
+
+```
+$ git clone git@github.com:username/conda-dot-org.git
+```
+
+#### 3. Creating a new branch
+
+Once you have cloned this repository, you now need to create a branch that will hold the
+work you are planning to contribute. You can do that with the following command:
+
+```
+$ git checkout -b my-new-contribution
+```
+
+Name the branch something related to the work you are doing. If you're working from
+an issue on our issue board, consider adding the issue number to the branch name.
+This can make it easier to find your work again later. For example:
+
+```
+$ git checkout -b 112-home-page-improvements
+```
+
+#### 4. Making edits and committing your changes
+
+Running the clone command (see step two) creates a copy of the repo in your file system. 
+You can then use a code editor of your choice (such as [VSCode][vscode-download], 
+[SublimeText][sublimetext-download], or [Atom][atom-download]) to make changes to the files.
+
+Once you have made the necessary edits, you can commit these edits by running the 
```

Comment:
At some point before this I expected an explanation of how to view your proposed update in a local instance.

Perhaps add some text here that points to #setting-up-a-local-version-of-the-site?  Or this can be in a later revision.

Reply:
Good idea. Done